Cabal/src/Distribution/Simple.hs

```diff
@@ -716,6 +718,19 @@ runConfigureScript verbosity backwardsCompatHack flags lbi = do
                ++ "If you are not on Windows, ensure that an 'sh' command "
                ++ "is discoverable in your path."
 
+-- | Convert Windows path to Unix ones
+toUnix :: String -> String
+#ifdef mingw32_HOST_OS
+toUnix s = let tmp = normalise s
+               (l:_, rest) = case splitDrive tmp of
+                               ([], x) -> ("/", x)
+                               x       -> x
+               parts = splitDirectories rest
+           in '/' : l : intercalate "/" parts
```

Comment:
Is there or should there be '/' between `l` and the rest?

Reply:
good catch, Just fixed it and the build warning.